images/win/scripts/Installers/Initialize-VM.ps1

```diff
@@ -30,6 +30,17 @@ function Disable-UserAccessControl {
     Write-Host "User Access Control (UAC) has been disabled."
 }
 
+function Disable-WindowsUpdate {
+    $AutoUpdatePath = "HKLM:SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate\AU"
+    If (Test-Path -Path $AutoUpdatePath) {
+        Set-ItemProperty -Path $AutoUpdatePath -Name NoAutoUpdate -Value 1
+        Write-Host "Disabled Windows Update"
+    } else {
+        Write-Host "Windows Update key does not exist"
+    }
+    
+}
+
 # Enable $ErrorActionPreference='Stop' for AllUsersAllHosts
 Add-Content -Path $profile.AllUsersAllHosts -Value '$ErrorActionPreference="Stop"'
 
```

Comment:
nitpick: replace setx -> [Environment]::SetEnvironmentVariable

Reply:
I have left this as it is because we use `setx` everywhere in code already